airmozilla/base/templates/500.html

```diff
@@ -4,16 +4,29 @@
     <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
     <title>Something went wrong!</title>
     <style type="text/css">
-      body {
-        font-family: "Open Sans Light","Lucida Sans","Lucida Grande","Lucida Sans Unicode",Verdana,sans-serif;
-        background-color: rgb(32, 39, 42);
-        color: rgb(255, 255, 255);
-      }
+      body {font-family: "Open Sans Light","Lucida Sans","Lucida Grande","Lucida Sans Unicode",Verdana,sans-serif;
```

Comment:
You can't use "Open Sans Light" because it's a webfont that needs to be downloaded. On the non-500 page this happens automatically because we're using tabzilla. 

I think the simplest solution would be to simply use "Verdana" and "sans-serif" like this:
```css
body {
    font-family: Verdana, sans-serif;
}
```